docs/develop/scripting-fundamentals.md

```diff
@@ -64,7 +64,7 @@ A range is a group of contiguous cells in the workbook. Scripts typically use A1
 
 Ranges have three core properties: values, formulas, and format. These properties get or set the cell values, formulas to be evaluated, and the visual formatting of the cells. They are accessed through `getValues`, `getFormulas`, and `getFormat`. Values and formulas can be changed with `setValues` and `setFormulas`, while the format is a `RangeFormat` object comprised of several smaller objects that are individually set.
 
-Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section of Using built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.
+Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.
```

Comment:
Also, I think these should be flipped in your PR.
```suggestion
Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md#working-with-ranges) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md) for more information on handling those arrays in the Office Scripts framework.
```

Reply:
Yup just saw that!